api/v1/utils/logs.js

```diff
@@ -25,25 +23,42 @@ function _filterBody(body, blacklist) {
   return _.omit(body, blacklist);
 }
 
-function _makeRequestMetadata(req) {
+function _acknowledgeRequest(req) {
   return {
     id: req.id,
     method: req.method,
-    url: req.originalUrl
+    url: req.originalUrl,
+    ip: req.ip
+  }
+}
+
+function _makeRequestMetadata(req) {
+  return {
+    id: req.id,
+    user: req.user ? req.user.get('id') : 'Unauthenticated',
+    adminOverRide: req.originUser ? req.originUser : 'No impersonation',
```

Comment:
What do you think about using `null` for the above values when they don't exist? Having string values seems a bit strange to me.